application/templates/datamanager/check-results.html

```
@@ -32,7 +32,26 @@ <h2 class="govuk-heading-m govuk-!-margin-top-3">Data has been checked</h2>
+      {% if window_start > 2 %}
+        {% set _ = pagination_items.append({"ellipsis": true}) %}
+      {% endif %}
+    {% endif %}
+    {% for p in range(window_start, page_number + 1) %}
+      {% set _ = pagination_items.append({"number": p, "current": p == page_number, "href": url_for('datamanager.check_results', request_id=request_id, page_number=p)}) %}
+    {% endfor %}
+    {% if has_next_page %}
+      {% set _ = pagination_items.append({"number": page_number + 1, "href": url_for('datamanager.check_results', request_id=request_id, page_number=page_number + 1)}) %}
+    {% endif %}
+    {% set pagination_params = {
+      "previous": {"href": url_for('datamanager.check_results', request_id=request_id, page_number=page_number - 1)} if page_number > 1 else none,
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)